examples/l1-resources/stream-connection.ts

```diff
@@ -25,7 +25,7 @@ export class CdkTestStack extends cdk.Stack {
 			instanceName: atlasProps.instanceName,
 			projectId: atlasProps.projectId,
 			connectionName: atlasProps.connectionName,
-			type: CfnStreamConnectionPropsType.CLUSTER,
+			type: atlasProps.type,
```

Comment:
Chose to reuse this example instead of creating a new one

Reply:
```suggestion
			type: CfnStreamConnectionPropsType.CLUSTER,
```
If we are re-using the example, I'd keep this as is so that it is clear that this is an example of stream connection type CLUSTER, since the params change based on the type of connection

Reply:
I probably should create another example then 🤔 ?
 Would you also expect `db_role_to_execute` for this CLUSTER example?

Reply:
Added examples: [5c87e0a](https://github.com/mongodb/awscdk-resources-mongodbatlas/pull/246/commits/5c87e0a10655f362178f778d426615984bea738e)

*See updated screenshots